azure-communication-ui/demo-app/src/chat/java/com/azure/android/communication/ui/chatdemoapp/ChatLauncherActivity.kt

```diff
@@ -256,4 +256,10 @@ class ChatLauncherActivity : AppCompatActivity() {
         val window: Window = this@ChatLauncherActivity.window
         window.navigationBarColor = ContextCompat.getColor(this@ChatLauncherActivity, R.color.white)
     }
+
+    override fun onDestroy() {
+        super.onDestroy()
```

Comment:
We should not close composite onDestroy as it could be due to device rotation, dark mode ON, or so.